### PR TITLE
[Addon] Add sts back to provider-alibaba

### PR DESCRIPTION
**Pull request by chivalryq**
### Description of your changes

Please ref https://github.com/oam-dev/kubevela/pull/2804, add optional tag for sts field for terraform-provider-alibaba

image

image

### Checklist

I have:
- [x] Title of the PR starts with OAM type (e.g. `[Workload]` or `[Trait]` or `[Scope]`).
- [ ] Updated/Added any relevant [documentation] and [examples].
- [ ] Unit/E2E Tests added.

### CODE REVIEW

File: addons/terraform-provider-alibaba/resources/parameter.cue [hunk 1]
```diff
@@ -1,5 +1,8 @@
 parameter: {
-	ALICLOUD_ACCESS_KEY: *"" | string
-	ALICLOUD_SECRET_KEY: *"" | string
-	ALICLOUD_REGION:     *"" | string
+	ALICLOUD_ACCESS_KEY: string
+	ALICLOUD_SECRET_KEY: string
+	// +optional
```
Comment (zzxwill) [inline addons/terraform-provider-alibaba/resources/parameter.cue:4]:
```suggestion
```

**Review by zzxwill — CHANGES_REQUESTED** [review]
Please append a snapshot to show it works on Dashboard. Thanks.

**Review by zzxwill — APPROVED** [review]

File: addons/terraform-provider-alibaba/resources/parameter.cue [hunk 2]
```diff
@@ -1,5 +1,7 @@
 parameter: {
-	ALICLOUD_ACCESS_KEY: *"" | string
-	ALICLOUD_SECRET_KEY: *"" | string
-	ALICLOUD_REGION:     *"" | string
+	ALICLOUD_ACCESS_KEY: string
+	ALICLOUD_SECRET_KEY: string
+	ALICLOUD_REGION: string
+	// +optional
+	ALICLOUD_SECURITY_TOKEN: *"" | string
```
Comment (barnettZQG) [inline addons/terraform-provider-alibaba/resources/parameter.cue:6]:
```suggestion
	ALICLOUD_SECURITY_TOKEN?: string
```

File: addons/terraform-provider-alibaba/resources/parameter.cue [hunk 3]
```diff
@@ -1,5 +1,7 @@
 parameter: {
-	ALICLOUD_ACCESS_KEY: *"" | string
-	ALICLOUD_SECRET_KEY: *"" | string
-	ALICLOUD_REGION:     *"" | string
+	ALICLOUD_ACCESS_KEY: string
```
Comment (barnettZQG) [inline addons/terraform-provider-alibaba/resources/parameter.cue:2]:
```suggestion
         +usage=Ensure that the AccessKey provided has reasonable permissions.
	ALICLOUD_ACCESS_KEY: string
```

**Comment by chivalryq** [comment]
This is not how this go, we should use cue's package to generate OpenAPI schema

**Pull request closed without merging**